Type mismatch: inferred type is Activity? but Activity was expected

**jitsi_meet plugin version**
<!-- e.g jitsi_meet: 3.0.0 -->
jitsi_meet:4.0.0

**Describe the bug**
<!-- A clear and concise description of what the bug is. -->
The error appeared immediately after upgrading to flutter version 3.0

**To Reproduce**
<!-- 
Steps to reproduce the behavior:

1. Upgrade flutter version to 3.0
2. Run flutter run 
3. The error will appear as show in screenshot below
-->

**Screenshots**
<!-- If applicable, add screenshots to help explain your problem. -->
![image](https://user-images.githubusercontent.com/29277048/168441804-ad0644a5-6c21-4684-84bd-3b3f9200b167.png)


**Flutter doctor -v**
<!-- Paste your output from flutter doctor -v here -->
![image](https://user-images.githubusercontent.com/29277048/168441853-1302d977-9c25-4d85-aca5-562d123c3a13.png)

**Web (please complete the following information):**
 - OS: [e.g. Mac, Windows, Linux]
 - Browser [e.g. chrome, safari]
 - Version [e.g. 22]

**Smartphone (please complete the following information):**
 - Device: Android
 - Emulator
 - OS: 
 - Version 23

**Additional context**
<!-- Add any other context about the problem here. -->

**Error Logs**
<!-- Paste flutter plus iOS/Android/Web debug logs here -->
```                  ^
e: /home/eli/snap/flutter/common/flutter/.pub-cache/hosted/pub.dartlang.org/jitsi_meet-4.0.0/android/src/main/kotlin/com/gunschu/jitsi_meet/JitsiMeetPlugin.kt: (66, 42): Type mismatch: inferred type is Activity? but Activity was expected

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':jitsi_meet:compileDebugKotlin'.
> A failure occurred while executing org.jetbrains.kotlin.compilerRunner.GradleCompilerRunnerWithWorkers$GradleKotlinCompilerWorkAction
   > Compilation error. See log for more details

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

````

--- a/jitsi_meet/android/src/main/kotlin/com/gunschu/jitsi_meet/JitsiMeetPlugin.kt
+++ b/jitsi_meet/android/src/main/kotlin/com/gunschu/jitsi_meet/JitsiMeetPlugin.kt
@@ -31,7 +31,7 @@ public class JitsiMeetPlugin() : FlutterPlugin, MethodCallHandler, ActivityAware
 
     private var activity: Activity? = null
 
-    constructor(activity: Activity) : this() {
+    constructor(activity: Activity?) : this() {
         this.activity = activity
     }
 

